rust/CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+TODO organize this
+
+Every md file has a html relative symlink to index.html:
+
+`ln -s index.html CONTRIBUTING.html`
+
+Project text doesn't link directly to documentation resources containing
+solutions - students should learn where to get the answers from the
+pre-requisites and lessons.
+
+Proect text may include inline links to pages that offer explanatios of terms
+and concepts.
+
+Keep the project summary (the **Task**, **Goals**, etc. text) synced between
+plan.md and the project description.
+
+Common sections of project descriptions:
+
+- The project summary (directly after h1)
+- "Introduction"
+- "Project spec"
+
+Headers do not capitalize every word; words after a colon
+are capitalized:
+
+> ## Project spec
+
+> ## Lesson: Tools and good bones
+
+In projects, be clear on when the student should start hacking, and what they
+should be hacking, by writing an imperative statement. Format that command in
+italics:
+
+> _Use [crates.io](https://crates.io) to find the documentation
+for the `clap` crate, and implement the command line interface
+such that the `cli_*` test cases pass._
```

Comment:
On the rendered page, the `_` in the inline code interrupts the `em` block. It's actually a [bug](https://github.com/markedjs/marked/issues/1410) of marked. Should we use html tag as a workaround or wait for them fixing it?